python/packages/autogen-agentchat/tests/test_group_chat.py

```diff
@@ -34,10 +34,10 @@
 from autogen_agentchat.teams._group_chat._swarm_group_chat import SwarmGroupChatManager
 from autogen_agentchat.ui import Console
 from autogen_core import AgentId, CancellationToken
+from autogen_core.models import ReplayChatCompletionClient
```

Comment:
```suggestion
from autogen_ext.models.replay import ReplayChatCompletionClient
```

Reply:
Do we want to move the replay client to Core? @jackgerrits

Reply:
I don't think so, they are all in a list in ext right now